src/patternfly/components/AppLauncher/app-launcher.scss

```diff
@@ -22,15 +25,44 @@
   --pf-c-app-launcher__toggle--disabled--Color: var(--pf-global--disabled-color--200);
   --pf-c-app-launcher__toggle--m-expanded--Color: var(--pf-global--Color--100);
 
+  // Menu input
+  --pf-c-app-launcher__menu-search--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-app-launcher__menu-search--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-app-launcher__menu-search--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-app-launcher__menu-search--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-app-launcher__menu-search--BottomBorderColor: var(--pf-global--BorderColor--100);
```

Comment:
Would you not use the divider component here?

Reply:
It's still experimental, we shouldn't be using it in components until it's promoted to a regular component. I have another issue open to update our examples in the components where we're using it now.

Reply:
Oh yep thats right!